modules/@apostrophecms/attachment/index.js

```diff
@@ -1100,7 +1070,8 @@ module.exports = {
           }
           return bulk.execute();
         }
-      }
+      },
+      ...require('./lib/legacy-migrations')(self)
```

Comment:
The migrations are methods, and there are other methods, so we use the spread syntax. Short of adding a new module format section just for migrations this seems like the way to go. `self` is passed in so we have access to the same scope as before.

Reply:
Sure, this looks good.

Reply:
Sure, this looks good. Thanks.